docs/UserGuide.md

```diff
@@ -1,33 +1,105 @@
-# User Guide
+# MTracker User Guide
 
 ## Introduction
 
-{Give a product intro}
+mTracker is a **command-line based trading journal interface** that **allows 
+investors and traders to store and view important trading related information** on their 
+shortlisted financial instruments for **reference and decision-making**. It **summarises 
+key details** into an **easy-to-read format and provides convenient lookups for trade setups**
+for busy individuals.
 
-## Quick Start
 
-{Give steps to get started quickly}
+## Quick Start
 
-1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+1. Ensure that you have Java `11` installed on your computer.
+2. Download the latest version of `MTracker` from [here](https://github.com/AY2122S1-CS2113T-T12-1/tp/releases).
+3. Copy the jar file to a folder that you want to run MTracker from.
+4. At the folder where you copied the jar file run the command `java -jar TP.jar` in terminal.
+5. If MTracker starts successfully, you should see the following output:
+```
+________________________________________________________________________________
+            _________                      __
+           |  _   _  |                    [  |  _
+ _ .--..--.|_/ | | \_| .--.  ,--.   .---.  | | / ] .---.  _ .--.
+[ `.-. .-. |   | |  [ `/'`\]`'_\ : / /'`\] | '' < / /__\\[ `/'`\]
+ | | | | | |  _| |_  | |    /| | |,| \__.  | |`\ \| \__., | |
+[___||__||__]|_____|[___]   \'-;__/'.___.'[__|  \_]'.__.'[___]
+
+Hello! I am mTracker, your personal assistant bot that
+helps you keep track of the markets.
+What should I do for you now? ☺
+________________________________________________________________________________
+```
 
 ## Features 
 
+## Notes about command format
+* Words in `UPPER_CASE` represent the parameters to be supplied by user. Below we have a list of parameters
+  that we would be referring to throughout this user guide.
+    * `NAME` represents the name of instrument to be specified by user.
+    * `PRICE` represents the price specified by the user.
+    * `DATE` represents the date specified by the user in `YYYY MM DD` format.
+    * `REMARKS` represents the remarks specified by the user.
+    * `INDEX` represents the index value of an instrument. The index value would correspond to the instrument's
+      position in the list.
```

Comment:
For this part for formatting of the parameters maybe NAME and PRICE is not really relevant as it is like common sense? But date remarks and index would clarify w the user so those are good to keep. Also can mention sentiments here